### DISCUSSION TITLE
Request for Feedback on GRASS GIS Documentation and Interest in GSoC Contribution

### DISCUSSION THREAD

**Discussion by rohannallamadge** [OSGeo/grass#4853] [General]
Hello
I hope you're doing well! I am currently working on improving a document as part of my contribution to the GRASS GIS Student Grants Program and would love your feedback on how I can enhance it further.  The document is located at this https://drive.google.com/file/d/1hksirjcocozzlDjSbyF8shGdQYPmK5oG/view?usp=sharing

**Areas Where I Would Appreciate Feedback:**
Clarity and Structure: Do the explanations and steps make sense? Is the document well-organized for someone new to GRASS GIS? Any suggestions on improving the flow?

**Code Examples**: Are the code examples clear and effective? Should I provide additional comments in the code or include more examples to improve understanding?

**Visuals and Results**: Do the images and visualizations effectively communicate the process and results? Would additional images or diagrams be helpful for understanding the workflows?

**Technical Accuracy**: Are there any parts that need clarification or improvement? Is the use of GRASS GIS modules and parameters accurately explained, or could any technical aspects be added or adjusted?

**Formatting and Style**: Is the document formatted in a way that aligns with the GRASS GIS documentation standards? Any suggestions on making it more user-friendly or visually appealing?

Additionally, I am also interested in contributing to Google Summer of Code (GSOC) in the upcoming cycle and would love any advice on how to better prepare for it. Do you think the work I’ve done so far aligns with the goals of GSOC, and what areas should I focus on to improve my chances of being selected?

I’d really appreciate any guidance you can provide to help me improve both the document and my overall contributions to GRASS GIS.

Thank you for your time and feedback!

Best regards,
Rohan 
[rohannallamadge@gmail.com](mailto:rohannallamadge@gmail.com)

**Comment by petrasovaa** [comment]
Couple points here:

* I strongly suggest not to use AI for crafting these questions, it tends to be vague and long and nobody wants to answer vague and long questions. 
* Sharing html file in google drive is not a good way. Creating a PR that modifies the documentation would be more appropriate.
* Please read contributing guidelines (CONTRIBUTING.md) including the linked documents about programming style and github contributions, this will answer some of your questions.
